docs/architecture/adr-006-poap-v2.md

```diff
@@ -0,0 +1,122 @@
+
+|      Variable       |  Type   | Description                                                                                                                                 |
+|:-------------------:|:-------:|:--------------------------------------------------------------------------------------------------------------------------------------------|
+|       `admin`       | Address | Identifies the user that controls the contract                                                                                              |
+|     `poap_name`     | String  | Identifies the name of the minted NFTs                                                                                                      |
+|    `poap_symbol`    | String  | Identifies the symbol of the minted NFTs                                                                                                    |
+| `poap_metadata_uri` | String  | Identifies the URI where users can view the associated metadata for the POAPs, ideally following the ERC-721 metadata scheme in a JSON file |
+
+The user can also provide the following:
+
+|        Variable        |   Type    |  Default  | Description                                                                                                           |
+|:----------------------:|:---------:|:---------:|:----------------------------------------------------------------------------------------------------------------------|
+|        `minter`        |  Address  | Undefined | Additional address that is allowed to mint tokens on behalf of other users                                            | 
+| `poap_is_transferable` |  Boolean  |  `False`  | Specifies whether each POAP can be transferred from one user to another                                               |
+|   `poap_is_mintable`   |  Boolean  |  `True`   | Indicates whether users can mint the POAPs                                                                            |
+| `poap_mint_start_time` | Timestamp | Undefined | Identifies the timestamp at which the minting of the POAP will be enabled. If not set, the minting is always enabled. |
+|  `poap_mint_end_time`  | Timestamp | Undefined | Identifies the timestamp at which the minting of the POAP will be disabled. If not set, the minting will never end.   |
+
+#### Execute
+
+The `MsgExecute` should allow the following operations:
+
+##### User operations
+
+Operations allowed for any user:
+
+1. Mint a new POAP if the minting conditions are met.
+2. Transfer a POAP they own to another user if transferring is allowed.
+3. Burn a POAP they own.
+
+##### Minter operations
+
+If set, the `minter` associated with the contract should be allowed to perform the following:
+
+1. Mint POAPs for a single user (`mintToSingleUser`).
+2. Mint POAPs for multiple users (`mintToManyUsers`).
+
+> **Note**  
+> The `minter` is not subject to the POAP minting conditions. This means that they act like and admin and can mint
```

Comment:
There is a typo: "[...] like **and** admin and can mint [...]" --> "[...] like **an** admin and can mint [...]".

Reply:
Fixed, thanks